azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -427,6 +431,16 @@ private void cleanOlderProjects() {
     }
   }
 
+  public void deleteDirectory(ProjectVersion pv) throws IOException {
```

Comment:
@HappyRay 
yes. didn't think about this. I have it `static` for now as per @chengren311 's suggestion. But we can always revert it back if the need arises.